WcaOnRails/app/models/post.rb

```diff
@@ -5,6 +5,21 @@ class Post < ActiveRecord::Base
   validates :body, presence: true
   validates :slug, presence: true, uniqueness: true
 
+  BREAK_TAG_RE = /<!--\s?break\s?-->/
```

Comment:
This only allows for 0 or 1 spaces, right? What about a non greedy match? Something like ` *?`